lib.sh

```diff
@@ -353,7 +353,10 @@ rh_finalize() {
     # Packaging cache is preserved across builds of container images
     $SUDO rm -f /etc/udev/rules.d/*-persistent-*.rules
     $SUDO touch /.unconfigured  # force firstboot to run
-    common_finalize
```

Comment:
Obviously this did not belong here

Reply:
It was intended to be there, but maybe no longer, I'll need to go look at the code to be sure...

Reply:
OIC, you're moving this and the debian one into `finalize()`, that's fine, thanks.